cmd/mockery.go

```diff
@@ -176,19 +198,18 @@ func (r *RootApp) Run() error {
 		log.Fatal().Msgf("Specify -dir or -srcpkg, but not both")
 	} else if r.Config.Name != "" {
 		recursive = r.Config.Recursive
-		if strings.ContainsAny(r.Config.Name, regexMetadataChars) {
-			if filter, err = regexp.Compile(r.Config.Name); err != nil {
-				log.Fatal().Err(err).Msgf("Invalid regular expression provided to -name")
-			} else if r.Config.FileName != "" || r.Config.StructName != "" {
-				log.Fatal().Msgf("Cannot specify --filename or --structname with regex in --name")
-			}
-		} else {
-			filter = regexp.MustCompile(fmt.Sprintf("^%s$", r.Config.Name))
+		names := strings.Split(r.Config.Name, ",")
```

Comment:
I think we can instead use https://pkg.go.dev/github.com/spf13/pflag#StringSlice to make the `--name` flag an explicit multi-value flag.